lms/envs/common.py

```diff
@@ -2050,3 +2050,25 @@
 SEARCH_ENGINE = None
 # Use the LMS specific result processor
 SEARCH_RESULT_PROCESSOR = "lms.lib.courseware_search.lms_result_processor.LmsSearchResultProcessor"
+
+######################## Leaner profile configuration ###########################
+
+# Default visibility level for accounts without a specified value
+# The value is one of: 'global', 'edx_users', 'private'
+DEFAULT_PROFILE_VISIBILITY = 'edx_users'
+
+# The list of fields that are shown on a shared learner profile
+SHARED_PROFILE_PUBLIC_FIELDS = [
```

Comment:
I find the "PUBLIC" here to be confusing. I would think of these as just the profile fields.